docs/tickets/310/ticket.md

```
@@ -0,0 +1,327 @@
+
+### **AS A** System Administrator
+
+**I WANT** to search and delete user accounts from CaTH
+
+**SO THAT** these users no longer have unrestricted access in CaTH
+
+
+### **ACCEPTANCE CRITERIA**
+* Only users with the **System Admin** role can access the System Admin Dashboard.
+* on the dashboard, when the system admin user clicks on the 'User Management' tab, then the system admin user is taken to the 'Find, update and delete a user' page and can view the list of CaTH users in a table that displays the following columns in sequential order 'Email', 'Role' and 'Provenance'. The 4th column without a title provides a link to 'Manage' each user provided in the table rows. Where there are several users, then a maximum of 25 users are displayed per page
+* A filter panel is provided on the left side of the page. The first section displays the 'Selected filters'. 2nd section displays 3 free text search fields. First is the 'Email' search field. When the correct search criteria is inputted and run, then the user list is updated to display matching results; however, where the wrong email is inputted, the following error message is displayed; '{**}There is a problem'. '{**}No users could be found matching your search criteria. Try adjusting or clearing the filters.' The second and third search fields are the 'User ID' and 'User Provenance ID' fields which display the following descriptive message beneath the field name and above the search bar 'Must be an exact match'. The 3rd section provides 2 check box sub-sections titled 'Role' which has the following options, 'Verified, CTSC Admin, Local[+] Admin and System Admin', and then 'Provenance' with the following options 'CFT IdAM and SSO'. [/+][-]Admin, CTSC Super Admin, Local Super Admin and System Admin', and then 'Provenance' with the following options 'B2C, CFT IdAM, Crime IdAM and SSO'.[/-]
```

markdownlint-cli2 (0.20.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "-"

(MD052, reference-links-images)